jetty-ee11/jetty-ee11-servlet/src/main/java/org/eclipse/jetty/ee11/servlet/ServletContextHandler.java

```diff
@@ -133,7 +133,7 @@
 public class ServletContextHandler extends ContextHandler
 {
     private static final Logger LOG = LoggerFactory.getLogger(ServletContextHandler.class);
-    public static final Environment ENVIRONMENT = Environment.ensure("ee11");
+    public static final Environment ENVIRONMENT = Environment.ensure("ee11", ServletContextHandler.class.getClassLoader());
```

Comment:
How is this possible?

The Environment is the classloader for the ServletContextHandler.
So if we reach this point, then ...

1. The Environment for "ee11" exists already.
2. The Environment classloader for "ee11" exists already too.

Right?

Reply:
Exactly! Either:
 a) The environment already exists (probably created by XML), in which case ensure checks that the classloader is the same one; OR
 b) The environment does not exist (embedded usage), in which case the environment is created with the loader that loaded the environment specific class.

Reply:
Saves creating the environments explicitly.

Consider:

```java
ClassLoader serverCL = Server.class.getClassLoader();

ClassLoader ee11CL = new URLClassLoader(...);

ContextHandler ee11Ctx = reflectionMagic(ee11CL);

server.setHandler(ee11Ctx);

server.start();
```

There is no need to create the `Environment` explicitly, and the ee11 environment will have the right ClassLoader.
Previously, it would have had the wrong ClassLoader.